easybuild/easyconfigs/c/CIRI/CIRI-2.0.6-intel-2017b-Perl-5.26.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'Tarball'
+
+name = 'CIRI'
+version = '2.0.6'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://sourceforge.net/projects/ciri/'
+description = "CircRNA Identifier. A de novo circular RNA identification tool"
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['CIRI_v%(version)s.zip']
+checksums = ['d0baef7e6b4589110f7cbc39f2ed168a4b86bc54d3b12360587d5336ae65e69b']
+
+dependencies = [('Perl', '5.26.0')]
+
+start_dir = 'CIRI_v%(version)s'
+
+sanity_check_paths = {
+    'files': ['CIRI%(version_major)s_manual.txt', 'CIRI%(version_major)s.pl'],
+    'dirs': ['data'],
+}
+
+modloadmsg = "To use CIRI, run 'perl $EBROOTCIRI/CIRI%(version_major)s.pl ...'\n"
```

Comment:
why not make it executable and add in `$PATH`?

Reply:
Because the software's documentation says "run `perl CIRI2.pl`, so people won't try to run `CIRI2.pl` directly, they'll try to run `perl CIRI2.pl` which will still fail even if the location of `CIRI2.pl` is in `$PATH`...